cmd/tailscale/cli/serve.go

```diff
@@ -66,10 +66,10 @@ EXAMPLES
   - To serve simple static text:
     $ tailscale serve https:8080 / text:"Hello, world!"
 
-  - To forward raw TCP packets to a local TCP server on port 5432:
-    $ tailscale serve tcp:2222 tcp://localhost:22
+  - To forward incoming TCP connections on port 2222 to a local TCP server on port 5432:
+    $ tailscale serve tcp:2222 tcp://localhost:5432
 
-  - To forward raw, TLS-terminated TCP packets to a local TCP server on port 80:
+  - To forward incoming TLS-terminated TCP connections to a local TCP server on port 80:
```

Comment:
I think the word order is wrong now for what this mode does.

Maybe: "To accept TCP TLS connections, terminate their TLS inside tailscaled, and then proxy to a local plaintext server on port 80, wrapping the plaintext response in TLS to the client:"

Or something. @shayne can help wordsmith.

Reply:
```suggestion
  - Accept TCP TLS connections (terminated within tailscaled) proxied to a local plaintext server on port 80. 
  ```

Reply:
Incorporated Shayne's suggestion.